modules/ROOT/pages/deployment/storage/posixfs.adoc

```diff
@@ -0,0 +1,104 @@
+* Only inotify and GPFS file system change notification methods are supported. +
+For mounted filesystems, `inotifywait` must be installed.
+* Versioning is not supported yet.
+* The spaces are folders in the filesystem and named after the UUID, not the real space name
+* No support for CephFS or other mounted filesystems like NFSv3 or SMB yet. +
+NFSv4.2 is working. This filesystem provides sufficient extended attribute capabilities. Check the documentation of your storage provider for details.
+* Postprocessing like antivirus scanning does not get triggered for file actions that are not initiated by Infinite Scale.
+* Though there are more options available, we recommend using a group based access for the shared storage. This means, that both Infinite Scale and the accessing users/services must use a common group. The default umask in the directory used has to allow group writing all over the place.
+
+== Setup
+
+Note that it is possible to use different storage drivers in the same Infinite Scale installation. For example, one is set up for one space running on PosixFS Storage Driver while all others run Decomposed FS with a private storage. This is an enhanced configuration and requires multiple instances of the xref:{s-path}/storage-users.adoc[Storage-Users] service running, individually configured. The details are not covered here.
+
+=== Prerequisites
+
+To use the PosixFS Storage Driver, the following prerequisites have to be fulfilled:
+
+* There must be storage available to store meta data and blobs, available under a root path.
+* When using inotify, the storage must be local on the same machine. +
+Network mounts require `inotifywait` to be installed (inotify does not work here).
+* The storage root path must be writeable and executable by the same user or group Infinite Scale is running under.
+* Infinite Scale must be at version 7 or higher.
+* You must use `nats-js-kv` as cache service, see xref:deployment/services/caching.adoc[Caching and Persistence] for more details.
+
+=== Configuration
+
+The following is an example configuration you can derive from with environment variables that configures Infinite Scale to use the PosixFS Storage Driver for all spaces. It works with Personal and Project Spaces. Use these settings in your deployment and replace `<local-path>` with your root entry point where Infinite Scale will access data.
+
+[source,yaml]
+----
+STORAGE_USERS_DRIVER="posix"
+STORAGE_USERS_POSIX_ROOT=<local-path>
+STORAGE_USERS_POSIX_WATCH_TYPE="inotifywait"
+STORAGE_USERS_ID_CACHE_STORE="nats-js-kv"
+STORAGE_USERS_ID_CACHE_STORE_NODES="localhost:9233"
+STORAGE_USERS_POSIX_USE_SPACE_GROUPS="true"          
+----
+
+=== GPFS Specifics
+
+When using GPFS as the underlying filesystem, the machine running the xref:{s-path}/storage-users.adoc[Storage-Users] service needs to have the GPFS filesystem mounted locally. The mount path is given to ocis as the `STORAGE_USERS_POSIX_ROOT` path.
```

Comment:
nitpick: the user doesn't necessarily have to set the mount path as the posix root. Instead the posix root is the directory where posixfs keeps it data. It may be the mount path or any subdirectory of it.